src/test/suite/fixtures/recorded/languages/html/takeBoundsBatt.yml

```diff
@@ -0,0 +1,31 @@
+languageId: html
+command:
+  version: 1
+  spokenForm: take bounds batt
+  action: setSelection
+  targets:
+    - type: primitive
+      modifier: {type: surroundingPair, delimiter: any, delimiterInclusion: excludeInterior}
+      mark: {type: decoratedSymbol, symbolColor: default, character: b}
+initialState:
+  documentContents: "<html>\r\n    <body>\r\n        <hr/>\r\n    </body>\r\n</html>"
```

Comment:
Mind fixing line endings and switching to "clear"?

Reply:
I swear those line endings are going to kill me. I keep changing them and changing them but vscode has no way of actually forcing line endings.